.github/workflows/haskell.yml

```diff
@@ -46,6 +46,19 @@ jobs:
         os: [ubuntu-latest]
 
     steps:
+
+    - run: |
```

Comment:
As discussed during the meeting let's add a condition for this to run only for nightly